README.md

```diff
@@ -1,3 +1,157 @@
-# Procedural-Ray-Tracing
-Repository for our Procedural Ray Tracing project, made by Y3 students at Breda University of Applied Science (former NHTV)
-More information: [Wiki](https://teamwisp.github.io/)
+# Wisp Rendering Framework
+We are a group of students from [Breda University of Applied Sciences]([https://www.buas.nl](https://www.buas.nl/)) working on a rendering framework using cutting-edge real-time ray-tracing technology from NVIDIA.
+
+## Table of contents
+
+- [Showcase](#showcase)
```

Comment:
I would put the showcase further down. If I was a developer looking at this page i would be annoyed at how far I have to scroll for the project description, features and the rest of the info.